src/components/header/Index.jsx

```diff
@@ -1,20 +1,26 @@
-/* eslint-disable no-unused-expressions */
 /* eslint-disable react/jsx-no-bind */
+/* eslint-disable react/jsx-indent */
+/* eslint-disable indent */
+/* eslint-disable no-unused-expressions */
```

Comment:
@olajide1234 why are we disabling this?

Reply:
Resolved. No rules are disabled in the page now.